apps/src/blockly/addons/cdoFieldImage.js

```diff
@@ -36,7 +36,7 @@ export default class CdoFieldImage extends GoogleBlockly.FieldImage {
   }
 
   shouldAllowImageChange() {
-    return this.allowImageChange;
+    return this.allowImageChange !== false;
```

Comment:
@code-dot-org/code-tools, the technique used to add the image seems to encounter `this.allowImageChange` while it's still `undefined`.  There's perhaps a better way to make sure it works?

Reply:
Discussed [here](https://codedotorg.slack.com/archives/C05DK21DAHK/p1697485044337719). Mind adding a comment that this can get called during object constructor, and an undefined value should therefore be treated as true?

Reply:
Done.  

Also, thanks for confirming that some poetry blocks get their quotation mark images back with this fix.  (I suspect they might have stopped working in https://github.com/code-dot-org/code-dot-org/pull/53051.)